.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 script: "./gradlew check integrationTest"
 
 jdk:
-  - openjdk6
+  - openjdk7
```

Comment:
The only risk is using APIs and language features not in 6, the android builds on jenkins should pick that up though